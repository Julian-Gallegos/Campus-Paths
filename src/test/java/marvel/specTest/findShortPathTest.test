# Test the parser on a graph that contains several nodes
# with two edges each.
# Then finds a path from one node to another node

LoadGraph heroes staffSuperheroes.tsv
FindPath heroes Notkin-of-the-Superhuman-Beard Grossman-the-Youngest-of-them-all