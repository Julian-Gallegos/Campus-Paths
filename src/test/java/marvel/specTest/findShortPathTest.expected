# Test the parser on a graph that contains several nodes
# with two edges each.
# Then finds a path from one node to another node

loaded graph heroes
path from Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all:
Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all via CSE331